backend/src/shared/services/googleapis.adapter.ts

```diff
@@ -0,0 +1,38 @@
+import { Injectable, Logger } from '@nestjs/common';
+import axios, { AxiosResponse } from 'axios';
+import { ApiConfigService } from './api-config.service';
+
+@Injectable()
+export class GoogleApisAdapter {
+  private readonly logger = new Logger(GoogleApisAdapter.name);
+
+  constructor(private apiConfigService: ApiConfigService) {}
+
+  async postToSafeBrowsing<T = any>(urlToCheck: string): Promise<T> {

```

Comment:
```suggestion
  async findUrlScore(url: string): Promise<GoogleSafeBrowsingResponse> {
```

Mà tốt nhất là để hàm này response number luôn